client/landing/jetpack-cloud/components/scan-history-item/style.scss

```diff
@@ -41,7 +47,7 @@
 		}
 	}
 
-	&__badge {
+	&__badge.badge {
```

Comment:
I noticed that this badge is not quite centered in height. 
Updating the line-height from 17 -> 14px fixed that for me. 

Before:
<img width="109" alt="Screen Shot 2020-04-10 at 10 16 55 AM" src="https://user-images.githubusercontent.com/115071/78975460-6ad30c80-7b14-11ea-935a-1535ac1286a9.png">

After:
<img width="100" alt="Screen Shot 2020-04-10 at 10 16 35 AM" src="https://user-images.githubusercontent.com/115071/78975432-5f7fe100-7b14-11ea-999a-46e83226cc34.png">

```suggestion
	&__badge.badge {
	line-height: 14px;
```